APISpec.md

```diff
@@ -97,3 +97,247 @@ Prerequisites are formatted as follows:
 ]` Means in order to take the course, you must have completed course id `2983` AND have completed `734` OR `1673`.
 
 ## Majors
+
+#### `GET /majors/`
+
+Returns a list of all majors.
+
+#### Response:
+
+```json
+[
+  {
+    "major_id": 34,
+    "name": "Computer Science",
+    "degree": "BS"
+  },
+  {
+    "major_id": 34,
+    "name": "Comparative Ethnic Studies",
+    "degree": "BA"
+  }
+  // ...
+]
+```
+
+#### `GET /majors/{major_id}`
+
+Returns data about a specific major.
+
+#### Response:
+
+```json
+
+{
+  "major_id": 34,
+  "name": "Computer Science",
+}
+
+```
+
+## Concentrations
+
+#### `GET /concentration/`
+
+Returns a list of all concentrations.
+
+#### Response:
+
+```json
+[
+  {
+    "major_id": 34,
+    "concentration_id": 56,
+    "name": "General Curriculum in Computer Science"
+  },
+  {
+    "major_id": 34,
+    "concentration_id": 57,
+    "name": "Artificial Intelligence and Machine Learning"
+  }
+  // ...
+]
+```
+
+#### `GET /concentration/{concentration_id}`
+
+Returns data about a specific concentration.
+
+#### Response:
+
+```json
+{
+  "major_id": 34,
+  "concentration_id": 56,
+  "name": "General Curriculum in Computer Science"
+}
+```
+
+## Users
+
+#### `POST /user/`
+
+Creates a new account.
+
+#### Request:
+
+```json
+{
+  "email": "example@gmail.com",
+  "first_name": "John",
+  "last_name": "Doe"
+}
+```
+
+#### Response:
+
+```json
+{
+  "user_id": 1
+}
+```
+
+#### `POST /user/{user_id}/course_history/{course_id}`
```

Comment:
Should we add `GET /user/{user_id}/course_history/` to return a list of courses the user has completed?

Reply:
See comment below for more details, but maybe this should be something like `POST /history/{course_id}`